packages/axelar-soroban-std/src/error.rs

```diff
@@ -111,7 +111,12 @@ macro_rules! assert_invoke_auth_ok {
         use soroban_sdk::IntoVal;
 
         let call_result = $client
-            .mock_auths($crate::mock_auth!($caller, $client, $method, $($arg),*))
+            .mock_auths(&[$crate::mock_auth!(
+                $client.env,
+                $caller,
+                $client.$method($($arg),*),
+                &[]
```

Comment:
```suggestion
                $client.$method($($arg),*)
```
remove the optional field now

Reply:
done